.github/workflows/main.yml

```diff
@@ -3,35 +3,36 @@ name: Unity Actions
 on:
   pull_request: {}
   push: {}
+  workflow_dispatch: {}
 
-env:
-  UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
-  UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
-  UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
-  PROJECT_PATH: .
-
+### Refer to https://game.ci/docs/github/getting-started
 jobs:
   checklicense:
```

Comment:
Should this be a job that we recommend? It feels like something you check once on setup rather than incur the cost for every CI run. It also doesn't capture the other complexities like using a serial number or licensing server instead of a license file.

Reply:
Right, I don't think we should recommend that they run this job every time. 
On the game.ci docs website, we have activation as a separate workflow. 
I think the goal for this repo is just to provide a very concise example, where a user could just clone the repo and quickly get a basic understanding of how to use the unity actions.